This patch fixes the header generation logic of the policy file
generation script which seems to be unable to deal with udev rules
shipped in newer systems.

Copyright 2022 林博仁(Buo-ren, Lin) <Buo.Ren.Lin@gmail.com>
SPDX-License-Identifier: MIT

--- policy/make-policy-file.in	2022-10-09 06:32:11.474304649 +0800
+++ policy/make-policy-file.in	2022-10-09 06:38:37.144661285 +0800
@@ -352,7 +352,7 @@
 get_header () {
     case "$MODE" in
 	udev)
-	    sed -n '/^#/!{ :t; /^SYSFS{id/q; /^ATTR{id/q; /^ATTRS{id/q; p; n; b t}' $1 \
+	    sed -n '/^#/!{ :t; /^SYSFS{id/q; /^ATTR{id/q; /^ATTRS{id/q; /^ATTRS{type/q; p; n; b t}' $1 \
 		| tac \
 		| sed -n -e '/^#/!{ :t; p; n; b t}' \
 		| tac \
